### DISCUSSION TITLE
What is the function of the Trackpad configuration in Window Throw?

### DISCUSSION THREAD

**Discussion by l-qing** [rxhanson/RectanglePro-Community#263] [Q&A] [answered]
Rectangle Pro has very powerful features, which I really like.
But there is one feature that I haven't figured out yet, and I don't know how to use it.

I cannot find the difference between setting it to "-" and "Rest 3 Touches, Remove All Except 1".

After pressing the modifier key, the only effect I noticed was that moving on the trackpad worked, but none of the other keys seemed to have any special functions.

![Shot 2023 04 24 at 20 49 21](https://user-images.githubusercontent.com/9499086/234000964-eb437685-0ba6-4483-80a9-3cb3a84cae4e.png)

Do we have any documentation to see a detailed description of your software? I couldn't find a detailed video on YouTube or official documentation.

**Comment by rxhanson** [comment, marked as answer]
I'm working on a video to demo this (videos are time consuming). 

The trackpad gesture is not used in conjunction with the modifier keys, it's by used by itself to trigger the window throw.

First, rest three touches. Remove two touches while keeping one on the trackpad, and then move the cursor with that one remaining touch. 

I think the video will make it simpler to understand, and to figure out if maybe it's not working for you due to some sort of bug.

**Reply by l-qing** [reply]
I tried using it according to your instructions and it worked out well. I didn't think of this kind of interaction method before, it's amazing.

**Comment by l-qing** [comment]
Your software allows resizing and moving windows anywhere on the screen using the trackpad. A similar software, [Swish](https://highlyopinionated.co/swish/), only allows this in specific areas.
I really like both of these software programs.

**Reply by rxhanson** [reply]
Thanks for the feedback. I've been asked a few times to implement Swish's gestures by users that had not tried the trackpad window throw, but I really prefer the window throw gesture. In my opinion, the window throw trackpad gesture is dramatically more efficient and flexible, but has a slightly higher learning curve.

**Reply by l-qing** [reply]
Thank you for your response.
I think both softwares have their own features, so there's no need to choose one over the other. I will continue to use both of them simultaneously.
